src/molecules/lists/ArticleList/index.js

```diff
@@ -17,15 +17,15 @@ function ArticleList( props ) {
   }), {});
 
   return (
-    <ul className={styles[type]} >
-      {data.map(item =>
+    <ul className={styles[type]}>
+      {data.map((item) => (
         <li className={styles['listItem']} key={item[itemProps.id]}>
           <a className={styles['link']} href={item[itemProps.link]}>
             <figure className={styles['articleWrapper']}>
               <img
                 className={styles['articleImg']}
                 src={item[itemProps.imageSrc]}
-                role='presentation'
+                alt=''
```

Comment:
why was `role` removed?

Reply:
(same as above)

the lint error was

```
Prefer alt="" over a presentational role. First rule of aria is to 
not use aria if it can be achieved via native HTML  
jsx-a11y/alt-text
```
so I replaced `role='presentation` with `alt=''`. But I just checked and it works if I keep both so I can add it back in if you'd prefer that.

Reply:
if it accomplishes the same thing idt we need both!

Reply:
discussed offline - keeping `alt=''`